lib/Minilla/Git.pm

```diff
@@ -70,5 +71,12 @@ sub git_submodule_files {
     return @files;
 }
 
+sub git_show_toplevel {
+    my $top_level = `git rev-parse --show-toplevel`
+        // errorf("Top-level git directory could not be found for %s\n", Cwd::getcwd());
```

Comment:
I think we should check `$?` here.

Reply:
That’s how I’d usually write it, but I wanted to try to match the style of the existing code.  Was there something in particular you had in mind to check `$?` for?  The [man page](https://perldoc.perl.org/perlop.html#Quote-Like-Operators) says that `` returns `undef` if the command fails, so I think this should do the same thing as checking that `$? =! 0`.

Reply:
`the command failed` means failing to execute command.
So `qx{command_does_not_exist}` returns undef, while `qx{perl -e 'exit 1'}` does not return undef.